l10n_it_corrispettivi_fatturapa_out/README.rst

```diff
@@ -0,0 +1,69 @@
+==========================================================
```

Comment:
regenerate readme, here links are still for 12.0

Reply:
how to regenerate? 😬

Reply:
The script is https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py, follow the instructions in https://github.com/OCA/maintainer-tools to install and https://github.com/OCA/maintainer-tools#readme-generator to use it.